test/expected/ddl_errors.out

```diff
@@ -18,14 +18,14 @@ ERROR:  integer dimensions require an explicit interval
 -- space dimensions require explicit number of partitions
 SELECT * FROM create_hypertable('"public"."Hypertable_1"', 'time', 'Device_id', chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
 ERROR:  invalid number of partitions for dimension "Device_id"
-SELECT * FROM create_hypertable('"public"."Hypertable_1_mispelled"', 'time', 'Device_id', 2, chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
-ERROR:  relation "public.Hypertable_1_mispelled" does not exist at character 33
-SELECT * FROM create_hypertable('"public"."Hypertable_1"', 'time_mispelled', 'Device_id', 2, chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
-ERROR:  column "time_mispelled" does not exist
+SELECT * FROM create_hypertable('"public"."Hypertable_1_misspelled"', 'time', 'Device_id', 2, chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
+ERROR:  relation "public.Hypertable_1_misspelled" does not exist at character 33
+SELECT * FROM create_hypertable('"public"."Hypertable_1"', 'time_misspelled', 'Device_id', 2, chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
+ERROR:  column "time_misspelled" does not exist
```

Comment:
Not sure about the goal here. If it's checking a `nonexistent` something, that'd probably be better than misspelling misspelled...